pkg/data/management/role_data.go

```diff
@@ -84,7 +84,9 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("management.cattle.io").resources("clustertemplates").verbs("*").
 		addRule().apiGroups("management.cattle.io").resources("globalroles", "globalrolebindings").verbs("*").
 		addRule().apiGroups("management.cattle.io").resources("users", "userattribute", "groups", "groupmembers").verbs("*").
-		addRule().apiGroups("management.cattle.io").resources("podsecuritypolicytemplates").verbs("*")
+		addRule().apiGroups("management.cattle.io").resources("podsecuritypolicytemplates").verbs("*").
+		addRule().apiGroups("management.cattle.io").resources("fleetworkspaces").verbs("*").
```

Comment:
Secret access is needed as UI has a feature to list available git clientSecret for user to select and add. I think it is fine as it is just list permission.

Reply:
I will remove this as it is giving secret content with `list`.